src/components/organisms/AdminDashboard/AdminDashboard.scss

```diff
@@ -1,18 +1,37 @@
 @import "scss/constants.scss";
 
-.admin-venue {
+.AdminDashboard {
   width: 100%;
-  height: 100%;
+  display: grid;
+  height: 100vh;
+  grid-template-columns: auto;
+  padding: $spacing--lg;
+
+  position: relative;
+  overflow-y: scroll;
 
   &__header {
     display: flex;
     align-items: center;
     justify-content: space-between;
+
+    &--left {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+    }
+  }
+
+  &__header-title {
+    font-size: $font-size--xxl;
+    font-weight: $font-weight--bolder;
+    align-self: center;
+    padding-right: $spacing--md;
   }
 
-  &__title {
-    font-size: 2em;
-    font-weight: bolder;
+  &__welcome-message {
+    font-size: $font-size--xxl;
+    font-weight: $font-weight--bolder;
```

Comment:
Probably best to substitute with existing `400`, `600` or `800` based `$font-weight--` constant. Makes it more deterministic.

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2278/commits/f423c3772cb4e9bb2a88f09ccc44a553786ca63d